react-hot-button-component/src/hot-button.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+class CustomButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { clicks: 0 };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  render() {
+    const clicks = this.state.clicks;
+    let id = '';
```

Comment:
This variable name is a bit confusing, it would be better to use something like `colorClass` or something similar.